### ISSUE TITLE
Midi over TRS.  

### ISSUE THREAD

**Issue by jimb-cc** [alf45tar/PedalinoMini#122]
Is Midi over TRS supported without extra hardware? As per the spec: https://www.midi.org/midi-articles/trs-specification-adopted-and-released

**Comment by alf45tar** [comment]
Yes it is. Electrical specification remains the same. It is just an adapter cable.
